assets/js/app.js

```diff
@@ -246,6 +246,28 @@ function setupDatePicker() {
   });
 }
 
+$(".fork-icon").on("click", function () {
+  let target_datetime = $(this).data("target-datetime");
+  date_picker = flatpickr("#restore_target", {enableTime: true, dateFormat: "Y-m-d H:i"})
+  date_picker.setDate(target_datetime, true);
+
+  $("#restore_target").addClass("animate-flash transition-colors duration-1000");
+  setTimeout(() => {
+    $("#restore_target").removeClass('animate-flash');
+  }, 2000);
+})
+
+$(document).on('change', '.connection-info-format-selector select, .connection-info-format-selector input', function() {
```

Comment:
Why do we add listener to whole document?

```suggestion
$(".connection-info-format-selector select, .connection-info-format-selector input").on('change', function() {
```